tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -407,4 +414,32 @@ _handle_ingress_aks() {
 
 	kustomize edit add resource ingress.yaml
 	popd
-}
\ No newline at end of file
+}
+
+
+# Implements the ingress handler for servernode
+# this is useful on kcli or anywhere where cluster IPs are accessible
+# from the testing machines.
+#
+_handle_ingress_nodeport() {
+	pushd "${COCO_KBS_DIR}/config/kubernetes/overlays"
+
+	cat > nodeport_service.yaml <<EOF
+# Service to expose the KBS on nodes
```

Comment:
@ldoktor the only thing that I would change on this PR is to contribute the nodeport_service.yaml to https://github.com/confidential-containers/trustee/tree/main/kbs/config/kubernetes . So it lives alongside the ingress deployment yaml for aks. What do you think?

Reply:
I thought about it but unlike AKS the nodeport service is quite hackish and mainly supported for debugging/testing purposes. I don't think it'd be wise to promote it as another "option".

Reply:
Good and fair point @ldoktor !